test/Microsoft.TemplateEngine.TestHelper/Commands/Assertions/CommandResultAssertions.cs

```diff
@@ -1,12 +1,15 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#if NET
```

Comment:
Where/when is this variable defined?

Reply:
This is defined when SDK-style projects are in use:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation
Basically it means "if on .NET", the reason for this is that base code is not available for full framework.